package exceptionexample;

public class ExceptionExample {

    public static void main(String[] args) {
        int data = 50 / 0;
        System.out.println("rest the code ....");
//        try {
//            int data = 50 / 0;
//        } catch (ArithmeticException e) {
//            System.out.println(e);
//        }
//        System.out.println("rest the code ....");
    }
}
